README.md

```diff
@@ -14,8 +14,10 @@ go get -u github.com/NYTimes/openapi2proto/cmd/openapi2proto
 There are 4 CLI flags for using the tool:
 * `-spec` to point to the appropriate OpenAPI spec file
 * `-annotate` to include (google.api.http options) for [grpc-gateway](https://github.com/gengo/grpc-gateway) users. This is disabled by default.
-* `-out` to have the output written to a file rather than `Stdout. Defaults to `Stdout` if this is not specified`
+* `-out` to have the output written to a file rather than `Stdout`. Defaults to `Stdout` if this is not specified.
 * `-indent` to override the default indentation for Protobuf specs of 4 spaces.
+* `-generateRpcs=false` to disable generation of rpcs. This is enabled by default.
```

Comment:
Do you have a a reason for not including the RPC definitions?

`protoc` ignores them unless the gRPC plugin is invoked and they do help define how the protobuf messages interact with one another.

Reply:
@jprobinson yes - we have use cases where we want to leverage protocol buffers outside of grpc, and so generating the rpc schemas and corresponding generated code will not be used. I agree this isn't a huge deal, but does help to remove clutter for our use case.

Reply:
I'm not 100% sure why, but something feels off about a camel case flag. I also wonder if we can make it cleaner by somehow making it `false` by default so just including some flag would drop the RPCs.

How about something like `-no-rpc` or `-no-grpc`?